action/protocol/context.go

```diff
@@ -129,6 +131,7 @@ type (
 		AddClaimRewardAddress                   bool
 		EnforceLegacyEndorsement                bool
 		EnableDynamicFeeTx                      bool
+		DisableBlobTransaction                  bool
```

Comment:
nit:
can we use `EnableBlobTransaction`? using `!DisableBlobTransaction` just feel weird to me